### ISSUE TITLE
about training loss

### ISSUE THREAD

**Issue by foralliance** [zisianw/FaceBoxes.PyTorch#83]
@zisianw  HI
I ran it exactly according to the training code. but I only have one card (1080TI), and the other parameters did not change at all. Training 28h.
My own training loss as follows:

```
2020-06-01 23:31:35,613:INFO:Epoch:1/300 || Epochiter: 1/403 || Iter: 1/120900 || L: 5.1744 C: 20.3799 || LR: 0.00100000 || Batchtime: 8.5759 s || ETA: 12 days, 0:00:31
2020-06-01 23:31:36,101:INFO:Epoch:1/300 || Epochiter: 2/403 || Iter: 2/120900 || L: 4.9724 C: 17.0202 || LR: 0.00100000 || Batchtime: 0.4392 s || ETA: 14:44:56
2020-06-01 23:31:36,637:INFO:Epoch:1/300 || Epochiter: 3/403 || Iter: 3/120900 || L: 4.7130 C: 14.2861 || LR: 0.00100000 || Batchtime: 0.4849 s || ETA: 16:16:57
2020-06-01 23:31:37,171:INFO:Epoch:1/300 || Epochiter: 4/403 || Iter: 4/120900 || L: 4.6357 C: 10.1130 || LR: 0.00100000 || Batchtime: 0.4832 s || ETA: 16:13:40
2020-06-01 23:31:37,685:INFO:Epoch:1/300 || Epochiter: 5/403 || Iter: 5/120900 || L: 4.3865 C: 8.5862 || LR: 0.00100000 || Batchtime: 0.4644 s || ETA: 15:35:48
2020-06-01 23:31:38,173:INFO:Epoch:1/300 || Epochiter: 6/403 || Iter: 6/120900 || L: 4.6417 C: 7.4876 || LR: 0.00100000 || Batchtime: 0.4380 s || ETA: 14:42:26
2020-06-01 23:31:38,742:INFO:Epoch:1/300 || Epochiter: 7/403 || Iter: 7/120900 || L: 4.5764 C: 6.7580 || LR: 0.00100000 || Batchtime: 0.5216 s || ETA: 17:30:59
2020-06-01 23:31:39,315:INFO:Epoch:1/300 || Epochiter: 8/403 || Iter: 8/120900 || L: 4.5351 C: 6.1112 || LR: 0.00100000 || Batchtime: 0.4555 s || ETA: 15:17:49
2020-06-01 23:31:41,792:INFO:Epoch:1/300 || Epochiter: 9/403 || Iter: 9/120900 || L: 4.3899 C: 5.7324 || LR: 0.00100000 || Batchtime: 2.4251 s || ETA: 3 days, 9:26:15
2020-06-01 23:31:42,196:INFO:Epoch:1/300 || Epochiter: 10/403 || Iter: 10/120900 || L: 4.2871 C: 5.4239 || LR: 0.00100000 || Batchtime: 0.3506 s || ETA: 11:46:21
2020-06-01 23:31:42,732:INFO:Epoch:1/300 || Epochiter: 11/403 || Iter: 11/120900 || L: 4.3746 C: 4.6924 || LR: 0.00100000 || Batchtime: 0.4889 s || ETA: 16:25:08
2020-06-01 23:31:43,167:INFO:Epoch:1/300 || Epochiter: 12/403 || Iter: 12/120900 || L: 4.3650 C: 4.5943 || LR: 0.00100000 || Batchtime: 0.3835 s || ETA: 12:52:42
2020-06-01 23:31:43,674:INFO:Epoch:1/300 || Epochiter: 13/403 || Iter: 13/120900 || L: 4.3817 C: 4.5847 || LR: 0.00100000 || Batchtime: 0.4606 s || ETA: 15:28:00
2020-06-01 23:31:44,178:INFO:Epoch:1/300 || Epochiter: 14/403 || Iter: 14/120900 || L: 4.5922 C: 4.5324 || LR: 0.00100000 || Batchtime: 0.4664 s || ETA: 15:39:47
2020-06-01 23:31:44,721:INFO:Epoch:1/300 || Epochiter: 15/403 || Iter: 15/120900 || L: 4.0729 C: 4.5218 || LR: 0.00100000 || Batchtime: 0.4953 s || ETA: 16:37:58
2020-06-01 23:31:45,249:INFO:Epoch:1/300 || Epochiter: 16/403 || Iter: 16/120900 || L: 4.7230 C: 4.3706 || LR: 0.00100000 || Batchtime: 0.4744 s || ETA: 15:55:52
2020-06-01 23:31:48,890:INFO:Epoch:1/300 || Epochiter: 17/403 || Iter: 17/120900 || L: 4.3539 C: 4.2943 || LR: 0.00100000 || Batchtime: 3.5956 s || ETA: 5 days, 0:44:07
2020-06-01 23:31:49,361:INFO:Epoch:1/300 || Epochiter: 18/403 || Iter: 18/120900 || L: 4.0115 C: 3.8395 || LR: 0.00100000 || Batchtime: 0.4222 s || ETA: 14:10:41
2020-06-01 23:31:49,843:INFO:Epoch:1/300 || Epochiter: 19/403 || Iter: 19/120900 || L: 4.1465 C: 3.9062 || LR: 0.00100000 || Batchtime: 0.4302 s || ETA: 14:26:40
2020-06-01 23:31:50,383:INFO:Epoch:1/300 || Epochiter: 20/403 || Iter: 20/120900 || L: 4.2397 C: 3.8839 || LR: 0.00100000 || Batchtime: 0.4904 s || ETA: 16:27:56
2020-06-01 23:31:50,907:INFO:Epoch:1/300 || Epochiter: 21/403 || Iter: 21/120900 || L: 4.1348 C: 3.8818 || LR: 0.00100000 || Batchtime: 0.4766 s || ETA: 16:00:12
2020-06-01 23:31:51,456:INFO:Epoch:1/300 || Epochiter: 22/403 || Iter: 22/120900 || L: 4.2044 C: 3.6742 || LR: 0.00100000 || Batchtime: 0.5001 s || ETA: 16:47:37
2020-06-01 23:31:51,966:INFO:Epoch:1/300 || Epochiter: 23/403 || Iter: 23/120900 || L: 4.3516 C: 3.8494 || LR: 0.00100000 || Batchtime: 0.4635 s || ETA: 15:33:50
....................................................
2020-06-03 03:10:04,192:INFO:Epoch:300/300 || Epochiter: 391/403 || Iter: 120888/120900 || L: 1.0186 C: 1.5184 || LR: 0.00001000 || Batchtime: 0.5481 s || ETA: 0:00:07
2020-06-03 03:10:04,650:INFO:Epoch:300/300 || Epochiter: 392/403 || Iter: 120889/120900 || L: 1.2653 C: 1.6692 || LR: 0.00001000 || Batchtime: 0.4121 s || ETA: 0:00:04
2020-06-03 03:10:05,650:INFO:Epoch:300/300 || Epochiter: 393/403 || Iter: 120890/120900 || L: 1.3480 C: 1.7201 || LR: 0.00001000 || Batchtime: 0.9479 s || ETA: 0:00:10
2020-06-03 03:10:06,072:INFO:Epoch:300/300 || Epochiter: 394/403 || Iter: 120891/120900 || L: 1.3920 C: 1.7500 || LR: 0.00001000 || Batchtime: 0.3724 s || ETA: 0:00:03
2020-06-03 03:10:06,473:INFO:Epoch:300/300 || Epochiter: 395/403 || Iter: 120892/120900 || L: 1.2428 C: 1.8737 || LR: 0.00001000 || Batchtime: 0.3517 s || ETA: 0:00:03
2020-06-03 03:10:06,862:INFO:Epoch:300/300 || Epochiter: 396/403 || Iter: 120893/120900 || L: 0.9238 C: 1.3636 || LR: 0.00001000 || Batchtime: 0.3386 s || ETA: 0:00:02
2020-06-03 03:10:07,221:INFO:Epoch:300/300 || Epochiter: 397/403 || Iter: 120894/120900 || L: 1.4052 C: 1.8867 || LR: 0.00001000 || Batchtime: 0.3073 s || ETA: 0:00:02
2020-06-03 03:10:07,538:INFO:Epoch:300/300 || Epochiter: 398/403 || Iter: 120895/120900 || L: 1.7991 C: 2.2399 || LR: 0.00001000 || Batchtime: 0.2653 s || ETA: 0:00:01
2020-06-03 03:10:07,841:INFO:Epoch:300/300 || Epochiter: 399/403 || Iter: 120896/120900 || L: 1.3118 C: 1.8016 || LR: 0.00001000 || Batchtime: 0.2534 s || ETA: 0:00:01
2020-06-03 03:10:08,106:INFO:Epoch:300/300 || Epochiter: 400/403 || Iter: 120897/120900 || L: 1.4301 C: 2.0517 || LR: 0.00001000 || Batchtime: 0.2131 s || ETA: 0:00:00
2020-06-03 03:10:08,371:INFO:Epoch:300/300 || Epochiter: 401/403 || Iter: 120898/120900 || L: 1.5059 C: 2.0308 || LR: 0.00001000 || Batchtime: 0.2136 s || ETA: 0:00:00
2020-06-03 03:10:08,637:INFO:Epoch:300/300 || Epochiter: 402/403 || Iter: 120899/120900 || L: 0.8883 C: 1.4389 || LR: 0.00001000 || Batchtime: 0.2136 s || ETA: 0:00:00
2020-06-03 03:10:08,743:INFO:Epoch:300/300 || Epochiter: 403/403 || Iter: 120900/120900 || L: 0.9817 C: 1.6208 || LR: 0.00001000 || Batchtime: 0.0920 s || ETA: 0:00:00
```
It is not known whether the above loss changes are normal. 
Thank you!
